### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by vishalg8454**
Added a Login page. The login page has following components:-

- Navbar
- Input fields
- Footer

image

### CODE REVIEW

File: pages/login-page/login-page.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+footer{
+    grid-row: 3/4;
+    background-color: var(--color-primary);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: var(--color-white)
+}
+.footer-list-container{
```
Comment (raghav6202) [inline pages/login-page/login-page.css:51]: Try to use only two names in naming classes

**Review by raghav6202 — APPROVED** [review]
Looks good to merge. Nice U.I 👍

File: pages/login-page/login-page.css (hunk 1 above)
Comment (vishalg8454) [reply pages/login-page/login-page.css:51]: Sure, will take care.

File: pages/login-page/login-page.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ranitasahaa1230) [inline pages/login-page/login-page.html:7]: You can add desc. in your meta tag like author name, page details, page name, etc

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to merge

File: pages/login-page/login-page.html (hunk 2 above)
Comment (vishalg8454) [reply pages/login-page/login-page.html:7]: Oh, new thing to me. Will learn and use in future.

**Pull request merged**